ntoskrnl/inbv/inbv.c

```diff
@@ -47,10 +47,26 @@ static PUCHAR ResourceList[1 + IDB_MAX_RESOURCE]; // First entry == NULL, follow
 /*
  * Change this to modify progress bar behaviour
  */
-#define ROT_BAR_DEFAULT_MODE  RB_PROGRESS_BAR
+#define ROT_BAR_DEFAULT_MODE    RB_PROGRESS_BAR
+
+/*
+ * Values for PltRotBarStatus:
+ * - PltRotBarStatus == 1, do palette fading-in (done elsewhere in ReactOS);
+ * - PltRotBarStatus == 2, do rotation bar animation;
+ * - PltRotBarStatus == 3, stop the animation thread.
+ * - Any other value is ignored and the animation thread continues to run.
+ */
+typedef enum _ROT_BAR_STATUS
+{
+    RB_FADEIN = 1,
+    RB_ANIMATE,
+    RB_STOP_ANIMATE,
+    RB_STATUSMAX
+} ROT_BAR_STATUS;
```

Comment:
If you have better names for all these, just let us know!

Reply:
Maybe RBS_* ?

Reply:
Good :)